### ISSUE TITLE
Nested double quotes don't work even when escaped

### ISSUE THREAD

**Issue by elimisteve** [vlang/v#5912]
**V version:** V 0.1.28 35df792 (from 2020.07.20; I didn't have this problem a couple days ago, so this is a recent regression, it seems)

**OS:** Debian 9.13

**What did you do?**

```
$ cat wontcompile.v
module main

fn main() {
    print(" \"Hello\" ")
}

$ v run wontcompile.v
```

**What did you expect to see?**

` "Hello" `
 
**What did you see instead?**

```
==================
/home/user/.cache/v/wontcompile.tmp.c: In function ‘main__main’:
/home/user/.cache/v/wontcompile.tmp.c:9403:21: error: expected ‘)’ before ‘Hello’
  print(tos_lit(" \\"Hello\\" "));
                     ^~~~~
/home/user/.cache/v/wontcompile.tmp.c:9403:26: error: stray ‘\’ in program
  print(tos_lit(" \\"Hello\\" "));
                          ^
/home/user/.cache/v/wontcompile.tmp.c:9403:27: error: stray ‘\’ in program
  print(tos_lit(" \\"Hello\\" "));
                           ^
/home/user/.cache/v/wontcompile.tmp.c: In function ‘_vinit’:
/home/user/.cache/v/wontcompile.tmp.c:9429:38: warning: integer constant is so large that it is unsigned
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.

If you were not working with C interop, please raise an issue on GitHub:

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang
```

**Comment by elimisteve** [comment]
' inside ', ' inside ", and " inside " all work; only " inside " (escaped double quotes inside outer double quote) fails.

**Comment by elimisteve** [comment]
Relatedly, probably,

`println(' It\"s OK ')`

also fails

**Comment by orneo1212** [comment]
Fixed by  #6474
